skymap_scanner/recos/dummy.py

```diff
@@ -69,3 +75,7 @@ def to_recopixelvariation(frame: I3Frame, geometry: I3Frame) -> RecoPixelVariati
             time=frame["Dummy_time"].value,
             energy=frame["Dummy_time"].value,
         )
+
+
+# Provide a standard alias for the reconstruction class provided by this module.
+RECO_CLASS: Final[type[RecoInterface]] = Dummy
```

Comment:
Since you're getting rid of the static resources, you can move this to the class itself as a getter/property. Then that'd force all new reco classes to implement it.

Reply:
> Since you're getting rid of the static resources, you can move this to the class itself as a getter/property. Then that'd force all new reco classes to implement it.

I am not sure I understand. The problem we are trying to solve is looking up the class name via the module name, so this has to be defined at the module level but outside the class.

Reply:
I acknowledge that this is not very elegant. We could consider a manually compiled lookup table in `__init__.py` but  I am not sure I like that more.